ref/feature/audit/description.adoc

```diff
@@ -0,0 +1,31 @@
+The Audit feature introduces an infrastructure that serves two purposes: \n\n\
+ - Confirming the effectiveness and integrity of the existing configuration \n\
+ - Identifying areas where improvement to the configuration may be needed  \n\n\
+
+The Audit feature can capture a range of auditable events, including \n\n\
+events related to authentication, authorization, and logout.
+//  - Basic authentication \n\
```

Comment:
Is the // intended? If it comments out the bullet I'd rather just remove the bullets.

Reply:
I left them in in case you wanted more detail adding back in but I'll remove them now.